src/test/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectApiRestClientTest.java

```diff
@@ -16,6 +16,12 @@
 
 package com.urswolfer.gerrit.client.rest.http.projects;
 
+import java.util.ArrayList;
```

Comment:
Please try to keep import ordering as it was before (probably in other files too).